test/Directory.Build.targets

```diff
@@ -6,5 +6,7 @@
     <PackageReference Remove="xunit" />
     <PackageReference Include="xunit.core" Version="$(XUnitVersion)" />
     <PackageReference Include="xunit.assert" Version="$(XUnitVersion)" />
+    <!-- Fix transitive System.Net.Http reference from xunit -->
```

Comment:
Weird that this is showing up here, it's already in main: https://github.com/dotnet/ef6/blob/main/test/Directory.Build.targets#L10